src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -212,10 +230,13 @@ public static String getNotificationMessage(String feedSourceName, Status status
         } else {
             // Processing did not complete. Depending on which sub-task this occurred in,
             // there may or may not have been a successful load/validation of the feed.
-            String errorReason = status.exceptionType != null ? String.format("error due to %s", status.exceptionType) : "unknown error";
-            LOG.warn("Error processing version {} because of {}.", feedVersion.id, errorReason);
-            message.append(String.format("While attempting to process a new feed version for %s, Data Tools encountered an " +
-                "unrecoverable error. More details: %s", feedSourceName, errorReason));
+            message.append(
+                String.format(
+                    "While attempting to process a new feed version for %s, Data Tools encountered an unrecoverable error. More details: %s",
```

Comment:
I think we should probably forego calling the application Data Tools and just use passive voice.

> ... an unrecoverable error was encountered

This is because there are different deployments of data tools that might go by different names for historical/project reasons.